Modules/Core/Common/src/itkFloatingPointExceptions_unix.cxx

```diff
@@ -136,7 +136,7 @@ void
 FloatingPointExceptions::Enable()
 {
   itkInitGlobalsMacro(PimplGlobals);
-#if defined(ITK_HAS_FPE_CAPABILITY) && !defined(__EMSCRIPTEN__)
+#if defined(ITK_HAS_FPE_CAPABILITY)
   itk_feenableexcept(FE_DIVBYZERO);
```

Comment:
What needs to be a no-op? `itk_feenableexcept` or something else?

Reply:
`itk_feenableexcept` is a noop, but we needed it to entered this code path.